Solution

ostapfriend opened the pull request:
https://ostapfriend.github.io/layout_miami/

ostapfriend commented:
https://ostapfriend.github.io/layout_miami/

ostapfriend commented:
https://ostapfriend.github.io/layout_miami/

ostapfriend commented:
> https://ostapfriend.github.io/layout_miami/

ostapfriend commented:
https://ostapfriend.github.io/layout_miamiMiami/

ostapfriend commented:
https://ostapfriend.github.io/layout_miami/

jstmpelowycz reviewed (changes requested):
Awesome 🔥
There are a few fixes left to apply:
	- replace the favicon with a vector image rather than a raster one
	- price label position is different from what is in the mockup, check it out
	- add hover effect for images in Compare Bikes and The Details sections
	- make sure all of your elements with effects on hover have the same transition duration
	- review [this](https://www.figma.com/file/Ic3SlZjkATYaS7uTifZAIk/BIKE?node-id=26%3A333) and compare it to your form fields
- the checklist: 4, 9, 14, 15

ostapfriend commented:
Fixed problem, but I have a few moments.
I didn't add vector image for bike, because this photo will be more bad than png.
I added target for link and fixed other problem.
Sorry,sometimes my english is not very good

 https://ostapfriend.github.io/layout_miami/

mykola-koval commented:
<img width="629" alt="image" src="https://user-images.githubusercontent.com/90685701/170995129-1326bfea-3062-4dfb-a7b0-d9730ed095b3.png">
this is what i see when clicking on links in header and footer.

they should not redirect to an external website, use anchor links (#)

mykola-koval commented:
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/90685701/170995296-6197a79c-ca7b-4e18-9fcc-d218cd11ff38.png">

mykola-koval reviewed (changes requested):
check out my comments and read through the checklist once again

ostapfriend commented:
https://ostapfriend.github.io/layout_miami/

ostapfriend commented:
https://ostapfriend.github.io/layout_miami/
